vic/drivers/image/src/get_global_param.c

```diff
@@ -777,6 +777,9 @@ get_global_param(FILE *gp)
             else if (strcasecmp("SKIPYEAR", optstr) == 0) {
                 sscanf(cmdstr, "%*s %hu", &global_param.skipyear);
             }
+            else if (strcasecmp("COORD_DIMS_OUT", optstr) == 0) {
+                sscanf(cmdstr, "%*s %zu", &options.COORD_DIMS_OUT);
+            }
```

Comment:
I'm also wondering if we can just determine this from the domain file.  It seems like that would be easier than making users define it in the global parameter file.

In the global parameter file, users should set the lat/lon coordinate names (e.g. xc or lon and yc or lat).  We should then be able to find out the shape/size of those variables and their dimensions.